internal/api/api.go

```diff
@@ -85,11 +85,22 @@ func (a *snykApiClient) GetUserMe() (string, error) {
 		return "", fmt.Errorf("error while fetching self data: %w", err) // Prioritize error
 	}
 
-	if len(selfData.Data.Attributes.Username) == 0 {
-		return "", fmt.Errorf("error while extracting user: missing property 'username'")
+	// according to API spec for get /self
+	// username is not a mandatory field, only name and email
+	// service accounts contain only name (name of the service account token) and org_context uuid
```

Comment:
**nitpick (non-blocking):** Maybe we could include a link to the public facing documentation for this endpoint

Reply:
good idea, done.